package.json

```diff
@@ -1,37 +1,65 @@
 {
-  "name": "frontend-scaffold",
-  "version": "0.1.0",
+  "name": "scaffold",
+  "version": "0.0.1",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
     "storybook": "start-storybook -p 6006 -c .storybook",
     "test": "jest",
-    "lint": "eslint 'src/**/*.{ts,tsx}'",
-    "pretty": "prettier --check 'src/**/*.{ts,tsx}' --write"
+    "lint": "eslint 'src/**/*.{ts,tsx}' --fix",
+    "pretty": "prettier --check 'src/**/*.{ts,tsx}' --write",
+    "format": "yarn pretty && yarn lint",
+    "cy:run": "yarn run cypress run",
+    "checks:run": "node ./.checks/run.js"
   },
   "dependencies": {
+    "@reach/alert": "^0.10.3",
```

Comment:
part of scaffold?

Reply:
Since it's some components are fairly common in our projects I thought it would be better to include some accessible primitives for them, see the readme.